db/seeds.d/62_shellhooks_proxy_feature.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+proxy_feature = Feature.where(name: 'Shellhooks').first_or_create
```

Comment:
@adamruzicka , does the name make difference, i.e. `'Shellhooks'` vs `'ShellHooks'`?

Reply:
It can be anything as long as its lower case version matches `shellhooks`

Reply:
We could maybe even split it into two words

Reply:
I asked because I don't really know if the name affects anything (like some magic that loads gem by it's name or constants). Looking at other examples of exposing, this looks OK and can be merged.

Reply:
Nothing crazy like that. It gets downcased, whitespace thrown out and then it gets matched against whatever the proxy returns as its features.

Reply:
Then I guess it's safe to merge.